### PR TITLE
Advanced data storage app

### PR DESCRIPTION
**Pull request by ravirajthedeveloper**

### CODE REVIEW

File: Freshdesk/v2/advanced_data_storage_app/app/js/app.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+/**
+ * Show modal to find link from the alias
+ */
+function openFindAliasModal() {
+    client.interface.trigger('showModal', {
+        title: 'Find Alias',
+        template: 'find_alias.html'
```
Comment (velmurugan-balasubramanian) [inline Freshdesk/v2/advanced_data_storage_app/app/js/app.js]: template/find_alias.html?

File: Freshdesk/v2/advanced_data_storage_app/app/js/app.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    });
+}
+
+/**
+ * Show modal to create new alias
+ */
+function openCreateAliasModal() {
+    client.interface.trigger('showModal', {
+        title: 'Add new Alias',
+        template: 'create_alias.html',
```
Comment (velmurugan-balasubramanian) [inline Freshdesk/v2/advanced_data_storage_app/app/js/app.js]: templates/create_alias.html?

File: Freshdesk/v2/advanced_data_storage_app/app/js/app.js (hunk 1 above)
Comment (ravirajthedeveloper) [reply Freshdesk/v2/advanced_data_storage_app/app/js/app.js]: Nope, it is a relative path. So, it looks for the template file from the current template file's location.

File: Freshdesk/v2/advanced_data_storage_app/app/js/app.js (hunk 2 above)
Comment (ravirajthedeveloper) [reply Freshdesk/v2/advanced_data_storage_app/app/js/app.js]: Nope again.

**Pull request merged**